dashboard/app/models/peer_review.rb

```diff
@@ -41,6 +41,40 @@ class PeerReview < ActiveRecord::Base
     escalated: 2
   }
 
+  def self.pull_review_from_pool(script, user)
+    # Find the first review such that meets these criteria
+    # Review is for this script
+    # I am not the submitter X
+    # Review status is nil
+    # Reviewer is nil or it has been assigned for more than a day
+    # Reviewer is not currently reviewing this level source
```

Comment:
We may also want the condition: "If the reviewer has a focus area containing this level, they must have submitted the level".